_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+- The default value of `sql.stats.cleanup.rows_to_delete_per_txn` has been increased to 10k to increase efficiency of the cleanup job for SQL statistics. [#97642][#97642]
+- The new new session setting `optimizer_use_improved_split_disjunction_for_joins` allows the optimizer to split disjunctions (`OR` expressions) in more `JOIN` conditions by building a `UNION` of two `JOIN` expressions. If this setting is true, all disjunctions in inner, semi, and anti `JOIN`s` will be split. Otherwise, only disjunctions that potentially contain an equijoin condition will be split. [#97696][#97696]
+- Builtins have been added for `tsvector`, `to_tsquery`, `phraseto_tsquery`, `plainto_tsquery`, which parse input documents into tsvectors and tsqueries respectively. The new `ts_parse` builtin is used to debug the text search parser. [#92966][#92966]
+- The new session variable `inject_retry_errors_on_commit_enabled` returns a transaction retry error if it is run inside of an explicit transaction when it is set to `true`. The transaction retry error continues to be returned until `inject_retry_errors_on_commit_enabled` is set to `false`. This setting allows developers to test their transaction retry logic. [#97226][#97226]
+- Previously, `ADD PRIMARY KEY NOT VALID` ignored the `NOT VALID` qualifier. This behavior was not compatible with PostgreSQL. CockroachDB now throws the error `PRIMARY KEY constraints cannot be marked NOT VALID`. [#97746][#97746]
+
+<h3 id="v23-1-0-alpha-5-operational-changes">Operational changes</h3>
+
+- The following cluster settings, which control rebalancing and upreplication behavior in the face of IO-overloaded storage, have been deprecated:
+  - `kv.allocator.l0_sublevels_threshold`
+  - `kv.allocator.l0_sublevels_threshold_enforce`
+  -
+  These cluster settings have been replaced by internal mechanisms. [#97142][#97142]
+
+- Max timeout-to-intent resolution has been added to prevent intent resolution from becoming stuck indefinitely and blocking other ranges attempting to resolve intents. [#91815][#91815]
+- Nodes are now considered suspect when re-joining a cluster and cannot accept lease transfers for one `server.time_after_store_suspect` window, which defaults to 30 seconds. [#97532][#97532]
+
+<h3 id="v23-1-0-alpha-5-command-line-changes">Command-line changes</h3>
+
+- The SQL shell (`cockroach sql`, `demo`) now supports the client-side commands `\l`, `\dn`, `\d`, `\di`, `\dm`, `\ds`, `\dt`, `\dv`, `\dC`, `\dT`, `\dd`, `\dg`, `\du` and `\dd` in a similar manner to PostgreSQL, including the modifier flags `S` and `+`, for convenience for users migrating from PostgreSQL. A notable difference is that when a pattern argument is specified, it should use the SQL `LIKE` syntax (with `%` representing the wildcard character) instead of PostgreSQL's glob-like syntax (with `*` representing wildcards). [#88061][#88061]
```

Comment:
```suggestion
- The SQL shell ([`cockroach sql`](cockroach-sql.html), [`cockroach demo`](cockroach-demo.html)) now supports the client-side commands `\l`, `\dn`, `\d`, `\di`, `\dm`, `\ds`, `\dt`, `\dv`, `\dC`, `\dT`, `\dd`, `\dg`, `\du` and `\dd` in a similar manner to PostgreSQL, including the modifier flags `S` and `+`, for convenience for users migrating from PostgreSQL. A notable difference is that when a pattern argument is specified, it should use the SQL `LIKE` syntax (with `%` representing the wildcard character) instead of PostgreSQL's glob-like syntax (with `*` representing wildcards). [#88061][#88061]
```